src/OpenTelemetry.Api/Trace/TelemetrySpan.cs

```diff
@@ -38,9 +38,9 @@ public abstract class TelemetrySpan
         public abstract bool IsRecording { get; }
 
         /// <summary>
-        /// Sets the status of the span execution.
+        /// Gets or sets the status of the span execution.
         /// </summary>
-        public abstract Status Status { set; }
+        public abstract Status Status { get; set; }
```

Comment:
there is a long discussion here https://github.com/open-telemetry/opentelemetry-dotnet/pull/464#discussion_r370081683 about this. There is no consensus as specification does not prescribe getter on status and it contradicts every other property on `TelemterySpan`

Reply:
I suspected that might get shot down. Reverted.